src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/ViewRenderer.js

```diff
@@ -35,31 +28,6 @@ class ViewRenderer extends React.Component<Props> {
         }, UPDATE_ROUTE_HOOK_PRIORITY);
     }
 
-    clearSidebarConfig() {
-        if (!this.hasSidebar()) {
-            sidebarStore.clearConfig();
```

Comment:
@wachterjohannes Any idea why this was implemented so complicated? I moved this call now to the `componentWillUnmount` of the `withSidebar` component, and I don't see a real difference in its behavior.